state/index.js

```diff
@@ -1,9 +1,17 @@
+import fetch from 'node-fetch';
+import Promise from 'bluebird';
```

Comment:
Promise should be native to Node 6+ (LTS release) is there a reason we don't use native promises?